pkg/apiserver/profiling/model.go

```diff
@@ -84,7 +84,7 @@ type TaskGroupModel struct {
 	ProfileDurationSecs    uint                          `json:"profile_duration_secs"`
 	TargetStats            model.RequestTargetStatistics `json:"target_stats" gorm:"embedded;embedded_prefix:target_stats_"`
 	StartedAt              int64                         `json:"started_at"`
-	RequstedProfilingTypes TaskProfilingTypeList         `json:"requsted_profiling_types"`
+	RequstedProfilingTypes TaskProfilingTypeList         `json:"requsted_profiling_types" swaggertype:"array,string"`
```

Comment:
To resolve: https://github.com/pingcap/tidb-dashboard/runs/4624353793?check_suite_focus=true

related issues:
- https://github.com/swaggo/swag/issues/679
- https://github.com/swaggo/swag/issues/817

solution:
- https://github.com/swaggo/swag#use-swaggertype-tag-to-supported-custom-type